airlock/business_logic.py

```diff
@@ -461,17 +461,15 @@ def request_filetype(self, relpath: UrlPath) -> None:
     def file_has_been_released(self, relpath: UrlPath) -> bool:
         return self.get_workspace_file_status(relpath) == WorkspaceFileStatus.RELEASED
 
-    def file_can_be_updated(self, relpath: UrlPath) -> bool:
+    def file_can_be_added(self, relpath: UrlPath) -> bool:
```

Comment:
We should be able to get rid of these methods on `Workspace` now that the enums are in their own module, and move the logic here into `policies.py` (they were only here before to avoid circular imports when importing `WorkspaceFileStatus` in `policies.py`)

Reply:
hah! doh! I did that originally, and then changed my mind & followed what you'd done elsewhere & moved them onto Workspace! I'll change.

Reply:
refactored all of these into the policies file